repo/packages/A/arangodb/0/config.json

```diff
@@ -0,0 +1,146 @@
+{
+  "type": "object",
+  "properties": {
+
+    "mesos": {
+      "description": "Mesos specific configuration properties",
+      "type": "object",
+      "properties": {
+        "master": {
+          "description": "The URL of the Mesos master. The format is a comma-delimited list of hosts like zk://host1:port,host2:port/mesos. If using ZooKeeper, pay particular attention to the leading zk:// and trailing /mesos! If not using ZooKeeper, standard URLs like http://localhost are also acceptable.",
+          "type": "string",
+          "default": "zk://master.mesos:2181/mesos"
+        }
+      },
+      "required": [
+        "master"
+      ]
+    },
+
+    "arangodb": {
+      "description": "ArangoDB specific configuration properties",
+      "type": "object",
+      "properties": {
+
+        "id": {
```

Comment:
@kensipe I've changed "id" to "framework-name" and removed the "id" also from the marathon.json